sidebars.js

```diff
@@ -19,8 +19,18 @@ module.exports = {
         "overview",
         {
           type: "link",
-          label: "PDF Download",
-          href: "/pdf",
```

Comment:
this should not be removed

Reply:
I think the PDF can be removed here, because it's not very useful in most cases..